llbuild.xcodeproj/project.pbxproj

```diff
@@ -1332,6 +1393,139 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		1484D1DC2094530300D3830F /* ui */ = {
+			isa = PBXGroup;
+			children = (
+				1484D1DD2094530300D3830F /* llbuildui */,
+				1484D1F12094530300D3830F /* README.md */,
+				1484D1F22094530300D3830F /* setup.py */,
+				1484D1F32094530300D3830F /* .gitignore */,
+			);
+			path = ui;
+			sourceTree = "<group>";
+		};
+		1484D1DD2094530300D3830F /* llbuildui */ = {
+			isa = PBXGroup;
+			children = (
+				1484D1DE2094530300D3830F /* trace.py */,
+				1484D1DF2094530300D3830F /* graphalgorithms.py */,
+				1484D1E02094530300D3830F /* database.py */,
+				1484D1E12094530300D3830F /* __init__.py */,
+				1484D1E22094530300D3830F /* orderedset.py */,
+				1484D1E32094530300D3830F /* model.py */,
+				1484D1E42094530300D3830F /* static */,
+				1484D1E62094530300D3830F /* app.py */,
+				1484D1E72094530300D3830F /* templates */,
+				1484D1F02094530300D3830F /* views.py */,
+			);
+			path = llbuildui;
+			sourceTree = "<group>";
```

Comment:
Is there a good reason to add these as file references instead of blue folders (which at least would stay in sync)?

Reply:
For item that are only loosely associated with the targets in Xcode I'd prefer blue folders for exactly that reason, they'd stay in sync.